### ISSUE TITLE
Live Preview Image Delay upon Scroll

### ISSUE THREAD

**Issue by danielrgjoseph** [kepano/obsidian-minimal#105]
Hi there,

I just updated to the latest version of minimal (V 4.4.1). I didn't experience this issue before, but when I have images on a page, and scroll down in live preview mode, there is a noticeable delay before the next image shows. When I try to scroll back up, there is once again another delay and until it finally appears, I just see a blank background.

**Comment by kepano** [comment]
This does not sound related to Minimal. Have you tried using the default theme? You can also try disabling plugins.

**Comment by danielrgjoseph** [comment]
Hi! I created a new vault and tested it out. The issue arises when "Readable Line Length" is disabled. When on, scrolling with images has no delay and is smooth, but when disabled, the images will disappear and reappear as I scroll.

I also just tested this out in another newly created vault with another theme, and it appears that this is an issue with live preview in general, not just the minimal theme. I have no clue to how to fix this.
